modules/gumgumBidAdapter.js

```diff
@@ -275,6 +275,9 @@ function buildRequests (validBidRequests, bidderRequest) {
     }
     if (params.inSlot) {
       data.si = parseInt(params.inSlot, 10);
+      if (params.sizes) {
+        data.bf = params.sizes.map((i) => `${i[0]}x${i[1]}`)
```

Comment:
If this is a new thing, if you could please add this at least to your .md and make sure it is known it MUST be an array of size arrays like so: `[[0, 1], [2, 3], [4, 5], [6, 7]]`

Else, make this code  catch bad input better so no exceptions are thrown?

Reply:
added to md file and catching bad input better

Reply:
Not sure you actually pushed the commits for this yet.

I do not see the `md` file update or any logic change.

Reply:
sorry, just pushed again